src/variorum/IBM/config_ibm.c

```diff
@@ -28,12 +28,10 @@ int set_ibm_func_ptrs(void)
         g_platform.variorum_print_power = p9_get_power;
         g_platform.variorum_print_power_limits = p9_get_power_limits;
         g_platform.variorum_cap_best_effort_node_power_limit =
-            p9_cap_node_power_limit;
+            p9_cap_and_verify_node_power_limit;
```

Comment:
I thought cap_and_verify was deprecated in this PR?

Reply:
This is the internal function for IBM which I didn't rename. Happy to rename it if you recommend. As we had discussed, we still want to verify that the cap was set, as a delay can be encountered sometimes.

Reply:
Ah ok, understand now.